### PR TITLE
Webatrice: show loading screen until protobuf initializes

### PR DESCRIPTION
**Pull request by seavor**
## Related Ticket(s)
- Fixes https://trello.com/c/B7MGtNHO/3

### CODE REVIEW

File: webclient/src/material-theme.ts [hunk 1]
```diff
@@ -193,11 +193,11 @@ export const materialTheme = createTheme({
       fontSize: 14,
       fontWeight: 'bold',
       lineHeight: 1.4,
-      color: '#9E9E9E',
+      color: palette.grey[500],
```
Comment (brentmclark) [inline webclient/src/material-theme.ts:196]: 😎

File: webclient/src/material-theme.ts [hunk 2]
```diff
@@ -193,11 +193,11 @@ export const materialTheme = createTheme({
...
     },
     subtitle2: {
       lineHeight: 1.4,
-      color: '#9E9E9E',
+      color: palette.grey[500],
```
Comment (brentmclark) [inline webclient/src/material-theme.ts:200]: 😎

**Review by brentmclark — APPROVED** [review]
LGTM :shipit:

**Comment by ZeldaZach** [comment]
Please resolve conflicts when you get a chance

**Pull request merged**